publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -245,6 +246,9 @@ private Publication fetchExistingPublication(RequestInfo requestInfo, SortableId
 
             throw new ForbiddenException();
         }
+        if (!isThesis(publication) && userCanEditAllNonDegreePublications(requestInfo)) {
```

Comment:
I know the !isThesis isn't needed. If we want to change something in the if-check above, we might allow editors to edit degrees as well.